packages/bff/src/utilities/errorMappers.ts

```diff
@@ -1,14 +1,19 @@
 import { constants } from "http2";
 import { ApiError, CommonErrorCodes } from "pagopa-interop-models";
 import { match } from "ts-pattern";
-import { ErrorCodes as LocalErrorCodes } from "../model/domain/errors.js";
+import { ErrorCodes } from "../model/domain/errors.js";
 
-type ErrorCodes = LocalErrorCodes | CommonErrorCodes;
+type BffErrorCodes = ErrorCodes | CommonErrorCodes;
```

Comment:
In the other services, we define the error codes type as `ErrorCodes`, there's a particular reason to rename this definition as `BffErrorCodes`?

Reply:
+1